python/ray/tests/test_autoscaler.py

```diff
@@ -1053,8 +1053,6 @@ def testConfiguresNewNodes(self):
 
     def testReportsConfigFailures(self):
         config = copy.deepcopy(SMALL_CLUSTER)
-        config["provider"]["type"] = "external"
-        config = prepare_config(config)
```

Comment:
removed as after we rewrite in prepare_config, the autofilling of instance resources fails as no module is provided. This prepare_config is redundant here too and we cannot have a type "external" without the appropriate module.